### DISCUSSION TITLE
Contribution Squares

### DISCUSSION THREAD

**Discussion by aaronnwg** [community/community#121877] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I have been pushing commits to the same 2 repositories for a relatively long time. These contributions have always shown up in my contribution squares. now it seems that starting at the beginning of April all of my contributions to these two repos stopped showing up in my squares. All of my contributions continue to show up and take effect in my repositories, but the squares remain empty. This was not an issue prior to April. My git flow has not changed. I have not forked the repositories. Im using the same email as I was before this problem. I would like to know how to resolve this issue so it doesn't continue and ideally get all my contributions from April to show up in the squares. It looks like I took a month off, but April was in fact a very busy month for me.

**Comment by MarcosApodaca** [comment]
Make sure that your contributions are not set to private or that you have not changed the visibility settings of the repositories.

**Reply by aaronnwg** [reply]
I have not changed anything from how it was originally configured when they were showing up.
